src/sidebar/SidebarItem.js

```diff
@@ -0,0 +1,22 @@
+import { useHistory, useLocation } from 'react-router-dom'
+import { MenuItem } from '@dhis2/ui'
+import { PropTypes } from '@dhis2/prop-types'
+import React from 'react'
+
+export const SidebarItem = ({ label, path }) => {
+    const history = useHistory()
+    const location = useLocation()
```

Comment:
You could use a slightly different hook:
```javascript
const MenuItemLink = ({ to, label }) => {
    const history = useHistory()
    const match = useRouteMatch(to)
    const onClick = () => history.push(to)
    return <MenuItem label={label} active={!!match} onClick={onClick} />
}
```

Reply:
Done, thanks